docs/configuration/configuration.md

```diff
@@ -15,7 +15,7 @@ Supported flags are as follows (result of running `./flagd start --help`):
   -k, --server-key-path string    Server side tls key path
   -a, --sync-provider-args        Sync provider arguments as key values separated by =
   -d, --socket-path string        Set the flagd socket path.
-  -f, --uri strings               Set a sync provider uri to read data from this can be a filepath or url. Using multiple providers is supported where collisions between flags with the same key, the later will be used.
+  -f, --uri strings               Set a sync provider uri to read data from this can be a filepath, url or reference to a kubernetes custom resource. Using multiple providers is supported, however if flag keys are duplicated across multiple sources it may lead to unexpected behavior.
```

Comment:
```suggestion
  -f, --uri strings               Set a sync provider uri to read data from this can be a filepath, url or reference to a kubernetes custom resource. Using multiple providers is supported, however if flag keys are duplicated across multiple sources the resulting behavior is undefined.
```